src/main/java/appeng/util/WirelessTerminalEvent.java

```diff
@@ -0,0 +1,30 @@
+package appeng.util;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.BiConsumer;
+
+import net.fabricmc.fabric.api.event.Event;
+import net.fabricmc.fabric.api.event.EventFactory;
+import net.minecraft.world.entity.player.Player;
+import net.minecraft.world.item.ItemStack;
+
+public class WirelessTerminalEvent {
+    public static final Event<BiConsumer<List<ItemStack>, Player>> EVENT = EventFactory.createArrayBacked(
```

Comment:
If you don't know what to write for the comment (I assume that's why it's a draft?), here's a suggestion:
```
Event fired when AE2 is looking for potential wireless terminal item stacks in a player inventory.
By default, AE2 only looks at the 36 usual slots of the player inventory, use this event to make AE2 consider more stacks.
The stacks don't need to contain a wireless terminal, AE2 will check after the event.
```

(you'll have to run spotless to format it)

Reply:
the main reason I converted it to a draft was because I hadn't tested it yet, and loading a world crashed because I forget to run datagen :D